cmd/appliance/upgrade/prepare.go

```diff
@@ -577,6 +582,10 @@ func prepareRun(cmd *cobra.Command, args []string, opts *prepareUpgradeOptions)
 				if !ok {
 					log.WithContext(ctx).Warning("no deadline in context")
 				}
+				// wait for appliance to be ready before preparing
+				if err := a.ApplianceStats.WaitForApplianceStatus(ctx, qs.appliance, appliancepkg.StatusNotBusy, nil); err != nil {
+					errs = multierr.Append(errs, err)
```

Comment:
why `errs` here? its not being used within this closure, I was expecting it to return here if error?

Reply:
ah, my mistake. not sure what I was thinking there.